### PR TITLE
Feature/submissions

### PR DESCRIPTION
**Pull request by yavetisyan**

### CODE REVIEW

File: src/pages/dashboard/SliderPerformers.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ const SliderPerformers = () => {
         renderArrow={myArrow}
       >
         {sliderItems.map((item) => (
-          <Box className="sliderBox" key={item.id}>
+          <Box className="slider__box" key={item.id}>
```
Comment (ArthurO2002) [inline src/pages/dashboard/SliderPerformers.tsx:110]: same issue

File: src/pages/events/RootEvents.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box>{children}</Box>}
+    </div>
+  )
+}
+
+const a11yProps = (index: number) => ({
```
Comment (ArthurO2002) [inline src/pages/events/RootEvents.tsx:35]: change function name

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react'
+import {Paper, Table, TableBody, TableCell, TableContainer, TableRow} from '@mui/material'
+import Report from './Report'
+import TableHeader from './TableHeader'
+
+export interface RowsInfo {
+  [keyof: string]: string | number
+}
+
+export type Order = 'asc' | 'desc'
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx:10]: move to types file

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    const stabilizedThis = array.map((el, index) => [el, index] as [T, number])
+    stabilizedThis.sort((a, b) => {
+      const order = comparator(a[0], b[0])
+      if (order !== 0) {
+        return order
+      }
+      return a[1] - b[1]
+    })
+    return stabilizedThis.map((el) => el[0])
+  }
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx:154]: move to utils

File: src/pages/events/eventsInfo/submission/Submissions.tsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  id: string
+  item: string
+  value: number
+}
+
+interface Arrow {
+  type?: string
+  onClick?: () => void
+  isEdge?: boolean
+}
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/submission/Submissions.tsx:24]: move to types

File: src/pages/sidebar/Sidebar.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ const Sidebar = () => {
             <AvatarIcon />
           </Avatar>
           <Typography className={classes.nameSurname}>Name Surname</Typography>
-          <NavLink style={{textDecoration: 'none'}} to="/">
+          <NavLink className="text__decoration_none" to="/">
```
Comment (ArthurO2002) [inline src/pages/sidebar/Sidebar.tsx:43]: issue with classes

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/pages/events/RootEvents.tsx (hunk 2 above)
Comment (yavetisyan) [reply src/pages/events/RootEvents.tsx:35]: Done

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx (hunk 3 above)
Comment (yavetisyan) [reply src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx:10]: Done

File: src/assets/style/submissions/submissions.scss [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  .rec.rec-carousel {
+    margin-top: $marginTop30;
+    margin-bottom: $marginBottom10;
+  }
+}
+
+.submission__quarterly {
+  margin: 0 20px;
```
Comment (ArthurO2002) [inline src/assets/style/submissions/submissions.scss:53]: use variables

File: src/pages/dashboard/EmptyDashboard.tsx [hunk 8]
```diff
@@ -18,7 +18,7 @@ const EmptyDashboard = () => {
         <Typography className={classes.dashText2}>
           To see dashboard information you need to set up evaluation event first.
         </Typography>
-        <Link className="text__decoration_none" to="/events-create">
+        <Link to="/" className="text-decoration__none">
           <Button variant="contained">CREATE EVENT</Button>
```
Comment (ArthurO2002) [inline src/pages/dashboard/EmptyDashboard.tsx:22]: not fixed

File: src/pages/events/eventsInfo/submission/submissionQuarterlyData/submissionNotFinished.ts [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    employeesName: 'Jenny Cooper',
+    employeesPosition: 'UI UX Desinger',
+    employerName: 'Anna Adams',
+    employerPosition: 'Project Manager',
+    date: 'Nov 9',
+    savedSubmission: false,
+  },
+]
+
+export default submissionNotFinished
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/submission/submissionQuarterlyData/submissionNotFinished.ts:18]: what is the point of this file, if it's used in multiple places then use enum

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/types/types.ts [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+  id: string
+  name: string
+  value?: number
+}
+
+export interface IArrow {
+  type?: string
+  onClick?: () => void
+  isEdge?: boolean
+}
```
[the remark this answers is not in the document]
Comment (yavetisyan) [reply src/types/types.ts:17]: Done

**Pull request merged**